components/Common/Tabs/index.module.css

```diff
@@ -0,0 +1,20 @@
+.tabsList {
+  @apply flex
+  gap-1
+  font-open-sans;
```

Comment:
I don't think we need to mark font-open-sans, it's already the default one, right?

Reply:
![image](https://github.com/nodejs/nodejs.org/assets/68445491/b9d91e6b-9c02-4297-abc8-45f58791b3b0)
Browser is overriding font-family for buttons

Reply:
`font-open-sans` can be at `.storybook/preview.tsx` file, it works too.

Reply:
hmmm why would it be overriding it?

Reply:
Just discovered the issue

![image](https://github.com/nodejs/nodejs.org/assets/68445491/dd1761df-9f14-4e95-80f1-e1eb3fe0c307)

Reply:
![image](https://github.com/nodejs/nodejs.org/assets/68445491/cb06f8a5-e9cc-40eb-afe5-391b065f15e8)

We've applied the `font-open-sans` to the body element, but it seems we only defined the variable `--font-open-sans` afterward. Could this be the potential issue?

Reply:
Let me do a hotfix

Reply:
Hotfix pushed, feel free to rebase.